src/liballoc/tests/btree/map.rs

```diff
@@ -753,10 +755,7 @@ fn test_bad_zst() {
 #[test]
 fn test_clone() {
     let mut map = BTreeMap::new();
-    #[cfg(not(miri))] // Miri is too slow
-    let size = 100;
-    #[cfg(miri)]
-    let size = 30;
+    let size = 12;
```

Comment:
This also affects non-Miri executions...

Maybe explain the magic constants? That applies everywhere, not just here. Anything you didn't pick at random (which is how I picked the previous numbers for Miri), please explain how you picked it.

Reply:
> This also affects non-Miri executions...

Yes, as suggested by @Mark-Simulacrum.

It's the minimum number of inserts to obtain height 2 (but not the minimum number of elements in a height 2 tree). I think we would benefit from some symbols to describe many of the magic numbers used in btree tests and/or have access to the B constant in BTree and/or the actual height of a tree. But I feel rather alone in this so it doesn't feel worthy of filing an issue and up to now I just try stuff in tests I contribute.

Reply:
Symbols would be great indeed, but just a comment like "obtain height 2 tree" or so would help already.